geometry_msgs/PointStamped targetPoint
---
bool pointReached